arraycontext/fake_numpy.py

```diff
@@ -75,7 +36,7 @@ def __init__(self, array_context):
         self.linalg = self._get_fake_numpy_linalg_namespace()
 
     def _get_fake_numpy_linalg_namespace(self):
-        return BaseFakeNumpyLinalgNamespace(self.array_context)
+        return BaseFakeNumpyLinalgNamespace(self._array_context)
 
     _numpy_math_functions = frozenset({
```

Comment:
What about this `_numpy_math_functions` list? Is this useful to keep in the `BaseFakeNumpyNamespace`? 

From what I remember, it's mostly a list of functions that loopy also supported and it's just used by the `pyopencl` implementation (pytato has its [own list](https://github.com/inducer/arraycontext/blob/5eec699eb1a81ef82879657b61731ea6a6441f11/arraycontext/impl/pytato/fake_numpy.py#L58-L60) of supported functions).

Reply:
> pytato has its own list of supported functions

That's actually sub-optimal. When we implememented `PytatoFakeNumpyNamespace` we didn't know about `BaseFakeNumpyNamespace._numpy_math_functions`. We should (I will :)) fix this at the PytatoArrayContext end. (Thanks!)